benchexec/tablegenerator/react-table/package.json

```diff
@@ -19,9 +19,41 @@
     "react-vis": "^1.11.7"
   },
   "scripts": {
+    "dependencies": "node scripts/dependencies.js",
     "start": "test -f src/data/dependencies.json || node scripts/dependencies.js && node scripts/start.js",
     "build": "node scripts/dependencies.js && node scripts/build.js",
-    "test": "test -f src/data/dependencies.json || node scripts/dependencies.js && node scripts/test.js"
+    "test": "test -f src/data/dependencies.json || node scripts/dependencies.js && node scripts/test.js",
+    "lint:fix:js": "prettier-eslint --write $(pwd)'/**/*.js'",
+    "lint:fix:css": "prettier --write $(pwd)'/**/*.css'",
+    "lint:fix:scss": "prettier --write $(pwd)'/**/*.scss'",
+    "lint:fix:json": "prettier --write $(pwd)'/**/*.json'",
+    "lint:fix": "npm run lint:fix:js && npm run lint:fix:css && npm run lint:fix:scss && npm run lint:fix:json",
+    "lint:js": "prettier --check $(pwd)'/**/*.js'",
+    "lint:css": "prettier --check $(pwd)'/**/*.css'",
+    "lint:scss": "prettier --check $(pwd)'/**/*.scss'",
+    "lint:json": "prettier --check $(pwd)'/**/*.json'",
+    "lint": "npm run lint:js && npm run lint:css && npm run lint:scss && npm run lint:json"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
```

Comment:
This partially duplicates the above rules, why? Also, it seems inconsistent, e.g., it runs `git add` only for `*.js`, and it uses `eslint-friendly-formatter` that is not referenced anywhere else.

Reply:
I also cleaned this up a bit.
Note that this `lint-staged` part is not a script but a configuration for the `lint-staged` package.
`eslint-friendly-formatter` is a terminal friendly wrapper for eslint. 
When you try to verify code with eslint in the console, you will see that it spams the terminal. This tool gets rid of that and makes it easier to find mistakes.